mod-orders/examples/bindPiecesCollection.sample

```diff
@@ -0,0 +1,26 @@
+{
+  "toBeBound": [
```

Comment:
We also need add these 5 fields to be able to create new item and connect pieces to that item
![image](https://github.com/folio-org/acq-models/assets/25097693/4099a8c4-c447-4ca8-9aec-66b4580a8760)

Reply:
thanks, added extra 5 fields along with poLineId